tests/test_events.py

```diff
@@ -180,64 +154,13 @@ def test_current_meeting_no_potentially_current(event):
     '''
     # Build an event outside of the "potentially current" timeframe.
     safety_meeting_info = {
-        'ocd_id': 'ocd-event/5e84e91d-279c-4c83-a463-4a0e05784b62',
+        'id': 'ocd-event/5e84e91d-279c-4c83-a463-4a0e05784b62',
         'name': 'System Safety, Security and Operations Committee',
-        'start_time': LAMetroEvent._time_from_now(hours=12),
+        'start_date': LAMetroEvent._time_from_now(hours=12),
     }
     event.build(**safety_meeting_info)
 
     current_meetings = LAMetroEvent.current_meeting()
 
     # Assert we did not return any current meetings.
     assert not current_meetings
-
```

Comment:
we don't have this attribute on the Events model anymore. It was a complicate bit of code, that we've handled upstream by making sure minutes are attache to the right meeting in the scraper code.